app/controllers/simulations_controller.rb

```diff
@@ -201,6 +201,12 @@ def mark_as_complete
             @experiment.progress_bar_update(@simulation_run.index, 'error')
           else
             @experiment.progress_bar_update(@simulation_run.index, 'done')
+            # Increment finished simulation counter in simulation manager record
```

Comment:
Ustawienie nie powinno być zależne od tego ifa. Jeżeli params[:status] == 'error', to i tak symulacja jest skończona, nie będzie powtarzana i tylko na pasku będzie czerwona. Jeżeli jej nie policzysz to nie doliczysz się do 100%.
Prawdopodobnie nie ma wtedy sensownych wyników, ale to nas nie interesuje.